pkg/services/policer/queue.go

```diff
@@ -14,8 +14,11 @@ func (q *jobQueue) Select(limit int) ([]*object.Address, error) {
 	// We can prioritize objects for migration, newly arrived objects, etc.
 	// It is recommended to make changes after updating the metabase
 
+	jobFilter := object.NewSearchFilters()
+	jobFilter.AddPhyFilter()
```

Comment:
As a small optimization we can for filters once and reuse them.

Reply:
Yeah, thought the same. I guess global var in policer package should work fine.